recipes/nim/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# based on https://github.com/bioconda/bioconda-recipes/tree/master/recipes/pb-falconc
+
+set -vexu -o pipefail
+
+pushd nim
+# inject compilers
+echo "gcc.exe = \"$CC\"" >> config/nim.cfg
+echo "gcc.linkerexe = \"$CC\"" >> config/nim.cfg
+echo "clang.exe = \"$CC\"" >> config/nim.cfg
+echo "clang.linkerexe = \"$CC\"" >> config/nim.cfg
+./build.sh
+bin/nim c koch
+./koch tools
+
+ls -larth
+ls -larth bin/
+ls -larth lib/
+ls -larth config/
+
+build_dir=$(pwd)
+
+mkdir -p "${PREFIX}"
+cd "${PREFIX}" || exit
```

Comment:
Why the `|| exit`?

Reply:
Yeah, it should be `exit 1`, but conda-build passes in `set -e` so this is not strictly needed

Reply:
Copy/paste error from elsewhere. Deleted.